### PR TITLE
fix: crash when socket destroyed during connecting

### PR DESCRIPTION
**Pull request by ngot**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

close https://github.com/eggjs/egg/issues/1641

### CODE REVIEW

File: lib/master.js [hunk 1]
```diff
@@ -134,8 +134,12 @@ class Master extends EventEmitter {
       // We received a connection and need to pass it to the appropriate
       // worker. Get the worker for this connection's source IP and pass
       // it the connection.
-      const worker = this.stickyWorker(connection.remoteAddress);
-      worker.send('sticky-session:connection', connection);
+      if (!connection.remoteAddress) {
+        connection.close();
```
Comment (popomore) [inline lib/master.js]: 能模拟吗

Comment (ngot) [reply lib/master.js]: 还没找到模拟的办法...

**Comment by popomore** [comment]
ci 我明天看看

**Comment by popomore** [comment]
https://github.com/eggjs/egg-cluster/pull/52

File: lib/master.js (hunk 1 above)
Comment (popomore) [reply lib/master.js]: 那覆盖率 ignore 一下

**Comment by popomore** [comment]
+ egg-cluster@1.12.5

File: lib/master.js (hunk 1 above)
Comment (atian25) [reply lib/master.js]: https://github.com/eggjs/egg/issues/3414

Comment (kiddnoke) [reply lib/master.js]: 这段逻辑是没什么错误的。 
当连接到来，看看远端地址是否存在，如果不存在，关闭连接
但是，关闭连接 没有close这个方法

**Pull request merged**